### DISCUSSION TITLE
OSRAM Smart+ Switch 4058075816459 - Misterious appearance

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#5536] [Devices]
After changing my coordinator there are some misterious appearance.
52 devices work fine, as with the old coordinator, when they where newly paired.

After paired with the new coordinator the Switch with 4 buttons fires every command six times.
The right 2 buttons work "normal" in this way.
The left 2 buttons switch EVERY device like Bulps, OnOff Switches from Sonoff or NoName into the state ON (left top) or OFF (left bottom) when they are pressed and the Smart+ Switch is NOT included in any flow.

How can i find the reason or debug this?
May be the device simply damaged?

My network is this:
- Raspberry PI 3B+
- Nodered 1.2.5
- Mosquitto MQTT 1.6.12
- Coordinator Texas Instruments CC2538 with CC2592 RF Amplifier from Tindie 
- Coordinator version: 20190608
- Zigbee2Mqtt: 1.16.1
- 50+ devices

**Comment by schamu** [comment]
I have this problem too, can anyone help?

**Reply by ghost** [reply]
I did not spend any more time to check it. I put this Smart+ Switch at the side.
I think there can be a reason in  the MQTT logics. One can use the /+/ or /# in topics to adress a lot of devices.

This week i got Tuya temperature sensors and recognize that they, if i press the send button, fires the temperature over 10 times in a second ... hm.
